addons/stock/models/product.py

```diff
@@ -517,6 +520,17 @@ def action_view_routes(self):
         action['domain'] = [('id', 'in', routes.ids)]
         return action
 
+    def action_create_inventory_adjustment(self):
```

Comment:
should this be public? if no, you could make product_id as an argument and simplify the calls

Reply:
yes, method should be public.